include/deal.II/grid/grid_tools.h

```diff
@@ -2358,10 +2358,11 @@ namespace GridTools
    * If no such relation exists then the returned std::optional object is empty
    * (i.e., has_value() will return `false`).
    *
-   * Here, two vertices <tt>v_1</tt> and <tt>v_2</tt> are considered equal, if
-   * $M\cdot v_1 + offset - v_2$ is parallel to the unit vector in unit
-   * direction @p direction. If the parameter @p matrix is a reference to a
-   * spacedim x spacedim matrix, $M$ is set to @p matrix, otherwise $M$ is the
+   * Here, two vertices <tt>v_1</tt> and <tt>v_2</tt> are considered equal (to
+   * the nearest tolerance), if $M\cdot v_1 + offset - v_2$ is parallel to the
```

Comment:
Could you update this formula to show exactly how it is related to the new tolerance parameter?